### PR TITLE
Some changes

### PR DESCRIPTION
**Pull request by omermorad**

### CODE REVIEW

File: dist/index.html [hunk 1]
```diff
@@ -5,6 +5,6 @@
     <title>Title</title>
 </head>
 <body>
-
+    This is just a dummy text in index html
```
Comment (GabiCtrlZ) [inline dist/index.html]: That's some dumb text

Comment (omermorad) [inline dist/index.html]: Disgusting

Comment (OsherSo) [inline dist/index.html]: Amazing !!!!!

Comment (omermorad) [inline dist/index.html]: asdasdasdasd

**Pull request merged**